bin/hdfcoinc/pycbc_fit_sngls_split_binned

```diff
@@ -129,6 +129,7 @@ for ex_p in extparams:
         params[ex_p] = np.array([trigf[args.ifo + '/template_duration'][ref][0]
                                  for ref in trigf[args.ifo + '/template_duration_template'][:]])
     else:
+        logging.info("Calculating " + ex_p + " from template parameters")
```

Comment:
Can we remove the logging statement on line 126 since now there is a statement for the parameter regardless of what it is?  (also, if it's template duration it isn't actually calculated.)